.github/workflows/template_release_drafter.yml

```
@@ -20,16 +20,39 @@ on:
 jobs:
   update_release_draft:
 
-    name: update release draft
+    name: Update Release Drafter
```

```suggestion
    name: Update Release Draft
```
We're updating the draft, not the drafter :D